tests/simple_server.py

```diff
@@ -70,6 +70,13 @@ def log_request(self, code='-', size='-'):
 # server on those Windows/Py2 to not fill the buffer.
 if six.PY2 and platform.system() == 'Windows':
   handler = QuietHTTPRequestHandler
```

Comment:
I don't think we still need this logic above if we now default to using `QuietHTTPRequestHandler`? Certainly we don't want two ways to change the HTTP Request Handler for this simple test server.

Reply:
I didn't want to remove this check so that this strange case for Windows is documented.

If I remove it now and someone change this piece of code again this bug can reappear.

Reply:
I can see the logic in that, we can either leave the comment and remove the code or consolidate this logic with your change. We should absolutely not set the handler in two places which can conflict with each other. As it stands if I explicitly set `use_quiet_http_request_handler = False` on Windows, I still get `QuietHTTPRequestHandler`.

Reply:
Yes, you are right. I will group those checks together so that we don't change the state in two places.

I am wondering which should be with higher priority: `use_quiet_http_request_handler` value or the check if we are on Windows?
I think that if we are on Windows should be with higher priority given that we have bug reported for that specific case.